### ISSUE TITLE
Django 1.5 Compatibility for Configurable User

### ISSUE THREAD

**Issue by trent** [caffeinehit/django-follow#18]
In Django 1.5, you can create a custom user model (https://docs.djangoproject.com/en/1.5//topics/auth/customizing/#substituting-a-custom-user-model) for the Auth contrib application. To reference this user in other apps, there is a new setting AUTH_USER_MODEL that should be used on any relationship fields. django-follow does not currently use this setting in the local follow.Follow model, you actually reference the django.contrib.auth.models.User object directly. This raises an error "...'user' defines a relation with the model 'auth.User', which has been swapped out. Update the relation to point at settings.AUTH_USER_MODEL." when running syncdb.

**Comment by flashingpumpkin** [comment]
Happy to accept pull requests. I'm a bit busy to add this myself right now and we've not got any Django 1.5 apps out there using django-follow.

**Comment by trent** [comment]
I'm working on a Django 1.5 app right now and happy to develop a patch.  How to you typically support existing installs?   Do you provide migrations, or just put that on the user to handle?

**Comment by flashingpumpkin** [comment]
I can include migrations into the code base - but that would only be for django-follow. I wouldn't want to mess with users' databases. :)

**Comment by trent** [comment]
of course.  i've actually decided to hold off on django 1.5 for my new project.  it seems many pluggable apps have not made considerations for configurable models and i have a lot of open tickets similar to this one elsewhere.  perhaps accommodating existing apps with ways to upgrade for this functionality is what is holding up the 1.5 release?  not sure, but i won't be putting any time against patching this ticket until the release is official.

**Comment by rchrd2** [comment]
Hello, I wanted to try this out in my Django 1.5 project so forked it and made these changes. Maybe I'm missing something, but the changes were really minimal. You can see the changeset here https://github.com/rcaceres/django-follow/commit/7a565351ccff14127cde274b6438f3abe9c17c1b

I have only tested with Django 1.5, but it worked for me. I can put a pull request if someone were to test this and confirm it works with older versions of Django.
